design/ImageBitmapToTexture.md

```diff
@@ -0,0 +1,36 @@
+# WebGPU + ImageBitmap
+
+```webidl
+dictionary ImageBitmapCopyView {
+    ImageBitmap imageBitmap;
+    GPUOrigin3D origin;
```

Comment:
This still references GPUOrigin3D, unlike the updated IDL, referencing the new GPUOrigin2D. :)

Reply:
Thanks for catching this!